docs/versioning/how-to-version-in-data-store.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+nav_order: 4
+title: How to create versions (Data Store) 
+has_children: false
+parent: Versioning
+---
+# How to create versions in the Data Store
```

Comment:
Do we need an explanation here/in the overview/in the FAQ about the DATA in the dataset i.e. the fact that it's a new storage space which is separately managed/accessed.

Reply:
maybe in the faq and in this page

Reply:
updated faq and in this page